### PR TITLE
Prosthetic stuff

### PR DESCRIPTION
**Pull request by MadmanMartian**
## About The Pull Request

Adds makeshift prosthetics

Adds the ability to craft the makeshift prosthetics

Adds the ability to craft meat hooks

Removes the useless subtype chooser for selecting an item to craft

Adds Makeshift prosthesis to character loadout

Adds Moebius full body prosthesis to character loadout, let's see how those go.

Adds department checks for loadout prosthesis

## Why It's Good For The Game

Sometimes you just can't wait in line for moebius to stop scratching their robo-butts to get a limb replacement.

Meat hooks were just a random item in maint, now they can be crafted

The subtype recipe picker was poorly implemented and useless, as all recipe subtypes would be in the list anyway.

## Changelog
:cl:
add: You can now craft makeshift prosthetics
add: Moebius can now opt in to a full body prosthesis through character loadouts.
/:cl:

### CODE REVIEW

**Review by genessee596 — CHANGES_REQUESTED** [review]
I'd like it if they were able to be chosen roundstart in the loadout, similar to other basic prosthetics. Useful for people who want to go all-in on Cybervag 2077.

If possible, it could give the player bonus loadout points in exchange for having such a flimsy prosthetic.

**Comment by MadmanMartian** [comment]
Already on it

File: code/__DEFINES/jobs.dm [hunk 1]
```diff
@@ -23,3 +23,5 @@
 #define DEPARTMENT_GUILD "Guild"
 #define DEPARTMENT_CIVILIAN	"Civilian"
 #define DEPARTMENT_CHURCH	"Church"
+
+#define ALL_DEPARTMENTS list(DEPARTMENT_COMMAND, DEPARTMENT_MEDICAL, DEPARTMENT_ENGINEERING, DEPARTMENT_SCIENCE, DEPARTMENT_SECURITY, DEPARTMENT_GUILD, DEPARTMENT_CIVILIAN, DEPARTMENT_CHURCH)
```
Comment (DarkLevelSS13) [inline code/__DEFINES/jobs.dm:27]: No newline at the end of file.

File: code/modules/organs/external/subtypes/robotic_types.dm [hunk 2]
```diff
@@ -149,3 +157,25 @@
...
+	default_description = /datum/organ_description/arm/left
+
+/obj/item/organ/external/robotic/makeshift/r_arm
+	default_description = /datum/organ_description/arm/right
+
+/obj/item/organ/external/robotic/makeshift/l_leg
+	default_description = /datum/organ_description/leg/left
+
+/obj/item/organ/external/robotic/makeshift/r_leg
+	default_description = /datum/organ_description/leg/right
```
Comment (DarkLevelSS13) [inline code/modules/organs/external/subtypes/robotic_types.dm:181]: No newline at the end of file.

File: code/datums/craft/recipes/misc.dm [hunk 3]
```diff
@@ -257,3 +257,41 @@
...
+		list(QUALITY_WIRE_CUTTING, 10, 10),
+		list(/obj/item/weapon/stock_parts/manipulator, 1),
+		list(QUALITY_PULSING, 10, 10),
+		list(/obj/item/weapon/tool/knife/hook, 1)
+	)
+	related_stats = list(STAT_MEC)
+
+/datum/craft_recipe/makeshift_arm/right
+	name = "Makeshift prosthetic right arm"
+	result = /obj/item/organ/external/robotic/makeshift/r_arm
```
Comment (DarkLevelSS13) [inline code/datums/craft/recipes/misc.dm:297]: No newline at the end of file.

**Review by DarkLevelSS13 — COMMENTED** [review]
No newline at the end of file.

**Review by genessee596 — APPROVED** [review]

**Review by TorinoFermic — APPROVED** [review]

**Review by mattroks101 — APPROVED** [review]

**Pull request merged**